draft-birkholz-cose-cometre-ccf-profile.md

```diff
@@ -185,12 +212,17 @@ This document requests IANA to add the following new value to the 'COSE Header P
 * Value type: `bstr`
 * Reference: This document
 
+* Label: TBD_3
```

Comment:
Polishing the registration section is still a todo, but that does not have to be part of this PR.

Reply:
To be done in separate PR.